src/style.css

```diff
@@ -1 +1,40 @@
-/* add styles here */
+.page_body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
The selector `.stars--0 .stars__star:nth-child(-n + 0)` will not select any elements because `nth-child(-n + 0)` is not a valid expression to select any children. If the intention is to not highlight any stars, this rule is unnecessary.